src/components/ui/dialog.tsx

```
@@ -0,0 +1,79 @@
+
+      if (open) {
+        dialogRef.current.showModal();
+      } else {
+        dialogRef.current.close();
+      }
+    }, [open]);
+
+    return (
+      <>
+        {open && (
+          <div className="fixed inset-0 z-40 bg-black/30" onClick={onClose} />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)